src/ex1/index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <title>Exercise 1</title>
-    <link rel="stylesheet" href="style.css" />
-</head>
-<body>
+   <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Weekend To-Do</title>
+      <link rel="stylesheet" href="style.css" />
+      <script
+         src="https://kit.fontawesome.com/dca2707a5a.js"
+         crossorigin="anonymous"
+      ></script>
+   </head>
 
-<h1>Todo list</h1>
+   <body>
+      <div class="card-container">
+         <div class="card">
+            <h1 class="app-title">Weekend To-Do</h1>
+            <p class="info">Get your tasks done before the weekend!</p>
+            <div class="container">
+               <input
+                  type="text"
+                  id="task-text"
+                  name="task-text"
+                  placeholder="Add your new to-do"
+               />
+               <button class="add-task">
+                  <i class="fa-solid fa-plus"></i>
+               </button>
+            </div>
+            <div class="tasks-and-clear-container">
+               <div class="all-tasks-container"></div>
+               <div class="footer-container">
+                  <p class="pending-tasks">You have 0 pending tasks</p>
```

Comment:
Consider: you could put the number in a span to make it easier to change via javascript (you won't have to maintain the same string in 2 places, you will have only one source of truth)
```suggestion
                  <p class="pending-tasks">You have <span id="tasks-counter">0</span> pending tasks</p>
```

Reply:
Added.